### DISCUSSION TITLE
Show what has been bought on Amazon?

### DISCUSSION THREAD

**Discussion by dekiesel** [firefly-iii/firefly-iii#8258] [Firefly III] [answered]
Hi,

At the moment it is pretty cumbersome to match transfers to Amazon to the products that were bought. Is it possible to connect my Amazon account to firefly so that it can match products to transactions so that I can improve my rules? At the moment I am tagging everything as `amazon`, but it could be `clothes`, `electronics`, `toys`...

**Comment by JC5** [comment, marked as answer]
Not built in, but you could program yourself using the API and webhooks.
